code/modules/food_and_drinks/drinks/drinks.dm

```diff
@@ -13,6 +13,7 @@
 	possible_transfer_amounts = list(5,10,15,20,25,30,50)
 	volume = 50
 	resistance_flags = NONE
+	you_drink_from_this = TRUE
```

Comment:
can you perhaps check the open_container flag instead of using a snowflake var

Reply:
> can you perhaps check the open_container flag instead of using a snowflake var

That's not a var that exists.

Reply:
its not a var, it's a flag on reagent containers, and its not verbatim either